src/ig/IGExporter.ts

```diff
@@ -48,6 +49,7 @@ export class IGExporter {
     this.addOtherPageContent(outPath);
     this.addImages(outPath);
     this.addIncludeContents(outPath);
+    this.addPredefinedResources(outPath);
```

Comment:
Probably inconsequential, but I feel like this might make more sense coming immediately after `this.addResources();`.